datafusion/core/tests/sqllogictests/test_files/select.slt

```diff
@@ -759,6 +759,64 @@ SELECT a FROM annotated_data_finite2
 0
 0
 
+# Below query should emit all the columns except a and b
```

Comment:
I am probably missing something, but these queries all appear to produce three output columns (a, b, and c), right? I am perhaps fundamentally misunderstanding what `EXCLUDE` is supposed to do or misunderstanding the comments

Reply:
This table has 5 columns `a0`, `a`, `b`,`c`,`d`.  The columns in the result are `a0`,`c`,`d`.
However, the column names are misleading.  Maybe we should change them also, also such as `a`, `b`,`c`,`d`, `e`

Reply:
This makes sense to me, the intent and behavior will be more clear. Thanks for the suggestion. I have changed the tests.